namespaces/api-description-namespace.md

```diff
@@ -95,6 +96,7 @@ The Resource representation with its available transitions and its data.
     - `href` (Templated Href) - URI Template of this resource.
     - `hrefVariables` (Href Variables) - Definition of URI Template variables used in the `href` property.
 - `content` (array)
+    - (Copy) - Textual information of the resource.
```

Comment:
"Textual information of the resource." does not sound right. What about something like "resource description's copy text" ?